src/gateway/how-kong-works/routing-traffic.md

```diff
@@ -495,87 +495,38 @@ This allow you to define two routes with two paths: `/service` and
 
 #### Using Regex in paths
 
-{{site.base_gateway}} supports regular expression pattern matching for an route's `paths` field
-via [PCRE](http://pcre.org/) (Perl Compatible Regular Expression). You can
-assign paths as both prefixes and Regex to a route at the same time.
-
-For a path to be considered as Regex, it must fall **outside** of the following regex:
+For a path to be considered a regular expression, it must be prefixed with a `~`: 
 
 ```
-^[a-zA-Z0-9\.\-_~/%]*$
+paths: ["~/foo/bar$"]
 ```
+Any path that isn't prefixed with a `~` will be considered plain text: 
 
-In other words, if a path contains any character that is **not** alphanumerical, dot (`.`),
-dash (`-`), underscore (`_`), tilde (`~`), forward-slash (`/`), or percent (`%`), then
-it will be considered a Regex path. This determination is done on a per-path basis
-and it is allowed to mix plain text and regex paths inside the same `paths` array of the same
-route object.
-
-For example, if we consider the following route:
-
-```json
-{
-    "paths": ["/users/\d+/profile", "/following"]
-}
 ```
-
-The following requests would be matched by this route:
-
-```http
-GET /following HTTP/1.1
-Host: ...
-```
-
-```http
-GET /users/123/profile HTTP/1.1
-Host: ...
+"paths": ["/users/\d+/profile", "/following"]
 ```
 
-The provided Regex are evaluated with the `a` PCRE flag (`PCRE_ANCHORED`),
-meaning that they will be constrained to match at the first matching point
-in the path (the root `/` character).
-
-**Note**: Regex matching is in general very fast, but it is possible to
-construct expressions that take a very long time to determine that they don't
-actually match.  This happens when the expression results in a "backtrace
-exponential explosion", which means that there's an astronomical number of
-tests to be performed before it's conclusive that they're all negative.
-If undetected, this could take hours to finish with a single regular expression.
-The [PCRE](http://pcre.org/) engine automatically detects most types of this
-and replaces with more direct approaches, but there are more complex expressions
-that could result in this scenario.
-
-To limit the worst-case scenario, {{site.base_gateway}} applies the OpenResty
-[`lua_regex_match_limit`](https://github.com/openresty/lua-nginx-module#lua_regex_match_limit)
-to ensure that any regex operation terminates in around two seconds in the
-worst case.  Apart from that, a smaller limit is applied to some "critical"
-regex operations, like those for path selection, in order to terminate them
-within two milliseconds, at most.
+For a deeper look into how the router processes regular expressions, read [performance considerations when using Expressions](/gateway/latest/key-concepts/routes/expressions/#performance-considerations-when-using-expressions).
```

Comment:
```suggestion
For more information about how the router processes regular expressions, see [performance considerations when using Expressions](/gateway/latest/key-concepts/routes/expressions/#performance-considerations-when-using-expressions).
```
Could just be a personal preference, "deeper look" sounds kind of marketing-y to me